base/stats.go

```diff
@@ -541,32 +527,32 @@ func (s *SgwStats) ClearDBStats(name string) {
 }
 
 func (d *DbStats) initCacheStats() {
-	dbName := d.dbName
+	labels := map[string]string{"database": d.dbName}
```

Comment:
Pulling this duplicated map creation into a method on `DbStats` to keep this in a single place (or make the database key a constant) would be my only nitpick on this PR, but otherwise looks fine to me.

Reply:
Went for constant option. Makes more sense imo